_data/team.yml

```
@@ -9,16 +9,20 @@
   title: Project Investment Fund Lead
   gravatar: fd6fd28a3a89cd5416d47ca691a08442
   email: chuck@codeforsanfrancisco.org
+- name: Dean Sofer
+  title: Project Success
+  gravatar: ae341f7eb365518d32b1b6bec287fa8b
+  email: dean@codeforsanfrancisco.org
 - name: Emily Wasserman
-  title: Organizer
+  title: Brigade Captain
   gravatar: 15a22b93c94ecd7c8cf880d8c5ac18ca
   email: emily@codeforsanfrancisco.org
 - name: Jason Lally
-  title: Lead Organizer
+  title: Organizer
   gravatar: b0d79f25e915d34f7f06907322ad8060
   email: jason@codeforsanfrancisco.org
 - name: Jesse Szwedko
-  title: Lead Organizer
+  title: Organizer
```

:+1: you can drop me from this page since I'm not active day-to-day.

---

```suggestion
  title: Web Admin
```

As you're current assistance has been appreciated

---

:smile: I appreciate the thought, but I still think it'd be more accurate for this to just reflect the current core team. I'm, of course, still happy to help through the transitions.